sass/_CategoryFacet.scss

```diff
@@ -0,0 +1,165 @@
+@import 'Variables';
+@import 'mixins/breadcrumb';
+@import 'mixins/_facetHeaderAnimation';
+@import 'bourbon/bourbon';
+
+@mixin facetValueLabelPadding {
+  padding: 5px 19px 5px 20px;
+}
+
+@mixin clickableCategoryFacetValue {
+  vertical-align: middle;
+  line-height: 22px;
+  cursor: pointer;
+  &:focus,
+  &:hover {
+    background-color: $color-blueish-white-grey;
+  }
+}
+
+.CoveoCategoryFacet {
+  &.coveo-hidden {
+    display: none;
+  }
+
+  & > .coveo-facet-values > .coveo-category-facet-value {
```

Comment:
I don't like those kind of precise selectors, they are very hard to customize. We should try to make things as flat as possible.

I understand it is "fighting" the base styling from the facet, but could we refactor facet styling or do it more intelligently in any way ?

This goes for this whole file, if we could make it so that "ideally" there is 0 nesting, only 1 rule for each styling we want to apply that would be perfect. I understand there will be exception to that rule, but it should be just that, an exception, and not the norm/standard.

Reply:
I removed this one since it wasn't even used anymore :neutral_face: As for the other ones, I reworked the way the html is built to be able to use simpler selectors.